File: ipman.shared/Entity/Vote.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ipman.shared.Entity
{
    public class Vote: EntityBase
    {
        public Guid PostID { get; set; }
        public Post Post { get; set; }
        public Guid UserAccountID { get; set; }
        public UserAccount UserAccount { get; set; }
        public DateTime CreatedUTC { get; set; }
        public bool IsUpTally { get; set; }
        public bool IsComment { get; set; }
    }
}

File: ipman/ClientApp/entity/Vote.ts
import { EntityBase } from "./EntityBase";
import Post from "./Post";
import UserAccount from "./UserAccount";

export default interface Vote extends EntityBase
{
    PostID: string;
    Post: Post;
    UserAccountID: string;
    UserAccount: UserAccount;
    CreatedUTC: Date | string;
    IsUpTally: boolean;
    IsComment: boolean;
}
